src/GraphQL.Tests/Types/TimeSpanMillisecondsGraphTypeTests.cs

```diff
@@ -65,13 +62,55 @@ public void coerces_TimeSpan_to_timespan()
             });
         }
 
+        [Theory]
+        [InlineData((int)5)]
+        [InlineData((long)7)]
+        public void parseliteral_to_timespan(object value)
+        {
+            CultureTestHelper.UseCultures(() =>
+            {
+                var expected = TimeSpan.FromMilliseconds(Convert.ToDouble(value));
+
+                IValue ast = value switch
+                {
+                    int i => new IntValue(i),
+                    long l => new LongValue(l),
+                    _ => null
+                };
+                var actual = _type.ParseLiteral(ast);
+
+                actual.ShouldBe(expected);
+            });
+        }
+
+        [Theory]
+        [InlineData((byte)1)]
+        [InlineData((sbyte)2)]
+        [InlineData((short)3)]
+        [InlineData((ushort)4)]
+        [InlineData((int)5)]
+        [InlineData((uint)6)]
+        [InlineData((long)7)]
+        [InlineData((ulong)8)]
+        public void parsevalue_to_timespan(object value)
```

Comment:
I don's see any negative tests for `ParseValue` and `ParseLiteral` for `TimeSpanMillisecondsGraphType` and `TimeSpanSecondsGraphType`. Either add them or let's add all lost test cases in separate PR with the help of codecov coverage colored map.

Reply:
#2355